File: PRAK602_2310817210022_Muhammad_Adh-Dhiya'Us_Salim.c
#include <stdio.h>

int main() {
    // Deklarasi variabel
    int jumlah_ruangan, i;

    // Meminta input jumlah ruangan
    scanf("%d", &jumlah_ruangan);

    // Deklarasi dan meminta input jumlah zetsu putih di setiap ruangan
    int zetsu_putih[jumlah_ruangan];
    for (i = 0; i < jumlah_ruangan; i++) {
        scanf("%d", &zetsu_putih[i]);
    }

    // Menghitung jumlah zetsu putih setelah pembelahan untuk setiap ruangan
    for (i = 0; i < jumlah_ruangan; i++) {
        zetsu_putih[i] = zetsu_putih[i] * (i + 1);
    }

    // Mencetak jumlah zetsu putih setelah pembelahan pada setiap ruangan
    for (i = 0; i < jumlah_ruangan; i++) {
        printf("%d ", zetsu_putih[i]);
    }

    return 0;
}

File: PRAK602_2310817210022_Muhammad_Adh-Dhiya'Us_Salim.py
# Meminta input jumlah ruangan
jumlah_ruangan = int(input())

# Meminta input jumlah zetsu putih di setiap ruangan
zetsu_putih = list(map(int, input().split()))

# Untuk setiap ruangan i, me-list, mau menghitung jumlah zetsu putih setelah membelah diri
for i in range(jumlah_ruangan):
 zetsu_putih[i] = zetsu_putih[i] * (i+1)

# Cetak jumlah zetsu putih setelah membelah diri pada setiap ruangan
for jumlah in zetsu_putih:
 print(jumlah, end=' ')
